### PR TITLE
add checkValidator class

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -68,6 +68,8 @@ dependencies {
     testCompile group: 'io.cucumber', name: 'cucumber-picocontainer', version: '4.7.1'
     compile group: 'net.masterthought', name: 'cucumber-reporting', version: '4.6.0'
     compile group: 'log4j', name: 'log4j', version: '1.2.17'
+    compile 'io.rest-assured:json-schema-validator:4.0.0'
+
```
Comment (aburgosu) [inline build.gradle]: delete white line

File: src/main/java/com/foundation/trello/util/SchemaValidator.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.foundation.trello.util;
+
+import io.restassured.response.Response;
+
```
Comment (aburgosu) [inline src/main/java/com/foundation/trello/util/SchemaValidator.java:4]: delete white line

File: src/main/java/com/foundation/trello/util/SchemaValidator.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+/**
+ * This class validates the schema of a request response.
+ * @author Josue Rodriguez
+ */
+public final class SchemaValidator {
+    /**
+     * This method constructor initializes the variables.
+     */
+    private SchemaValidator() {
+
```
Comment (aburgosu) [inline src/main/java/com/foundation/trello/util/SchemaValidator.java:16]: delete white line

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 4]
```diff
@@ -4,12 +4,15 @@
...
 import com.foundation.trello.model.request.RequestManagerAbstract;
+import com.foundation.trello.util.SchemaValidator;
+import cucumber.api.java.en.And;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 import io.restassured.response.Response;
 import org.testng.asserts.SoftAssert;
 
+
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java]: Delete unnecessary line.

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 5]
```diff
@@ -73,4 +76,13 @@ public void getStatusCodeAsResponse(int statusCode) {
         SoftAssert softAssert = new SoftAssert();
         softAssert.assertEquals(statusCode, response.getStatusCode());
     }
+
+    /**
+     * This method verifies that the schema response to the requirement is correct.
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java:80]: Please add a line.

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java (hunk 4 above)
Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java]: I fixed it.

File: build.gradle (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply build.gradle]: I fixed it.

File: src/main/java/com/foundation/trello/util/SchemaValidator.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/foundation/trello/util/SchemaValidator.java:4]: I fixed it.

File: src/main/java/com/foundation/trello/util/SchemaValidator.java (hunk 3 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/foundation/trello/util/SchemaValidator.java:16]: I fixed it.

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java (hunk 5 above)
Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java:80]: I fixed it.

**Review by MadayAlcala — APPROVED** [review]

**Review by aburgosu — APPROVED** [review]

**Pull request merged**